pkg/controller/userreport/controller.go

```diff
@@ -61,10 +60,7 @@ func New(locales *i18n.LocaleMap, cacher cache.Cacher, cfg *config.RedirectConfi
 
 	localCache, _ := memcache.New(30 * time.Second)
 
-	httpClient := &http.Client{
-		Timeout:   10 * time.Second,
-		Transport: project.DefaultHTTPTransport(),
-	}
+	httpClient := controller.TracedHTTPClient(10 * time.Second)
```

Comment:
This will propagate the trace ID to external clients (used for webhook)

Do we want to do that?

Reply:
I was interested in getting the request ID, but I realize that might be a bad idea now. Reverting.